.github/actions/code-coverage/action.yml

```diff
@@ -0,0 +1,77 @@
+name: Code Coverage Report
+description: 'Adds a comment to PRs with code coverage'
+
+inputs:
+  coverage-counter-type:
```

Comment:
This breaks our convention of underscores for properties. Looks like a mixup of the kover action property and ours - from below: `coverage-counter-type: ${{ inputs.coverage_counter_type }}`

Reply:
Yes I was originally copy/pasting from the action we are using until I realized the conventions don't match. Must have missed switching one. Good to know an action does not fail when referencing an undeclared input  😬 .